xml/System.Security.Cryptography.Xml/RSAKeyValue.xml

```diff
@@ -264,7 +264,17 @@ Based upon [The RSAKeyValue Element](https://www.w3.org/TR/2002/REC-xmldsig-core
           ]]></format>
         </remarks>
         <exception cref="T:System.ArgumentNullException">The <paramref name="value" /> parameter is <see langword="null" />.</exception>
-        <exception cref="T:System.Security.Cryptography.CryptographicException">The <paramref name="value" /> parameter is not a valid <see cref="T:System.Security.Cryptography.RSA" /> key clause XML element.</exception>
+        <exception cref="T:System.Security.Cryptography.CryptographicException">The <paramref name="value" /> parameter is not a valid <see cref="T:System.Security.Cryptography.RSA" /> key clause XML element.
+
+-or-
+
+The XML has the incorrect schema.
+
+-or-
+
+The DSA parameters are invalid.
```

Comment:
Same comment as with DSA.  Though if we keep the two new sentences here this one should say "RSA" instead of "DSA" :smile:.